### ISSUE TITLE
Alberto Daniel Hill longstanding issue with local law enforcement.

### ISSUE THREAD

**Issue by sickcodes** [disclose/research-threats#41]
I hope Alberto can enlighten us in a few short sentences!

**Comment by PinkDev1** [comment]
Is this about the same Alberto Daniel Hill who runs the DARKWEB.TODAY podcast? Please provide more information about this

**Comment by sickcodes** [comment]
Yup, Alberto's seen this, I'll have a chat with him and get a short "thing" hopefully some pics from the archives for the folder that the police seized, never charged, or never returned, etc.

**Comment by nofear75** [comment]
In 2014, Alberto Daniel Hill, an expert in cybersecurity, found a security issue in a medical provider’s website. In reporting the issue, it led him to become the first person imprisoned in Uruguay for a computer-related crime—a crime he didn’t commit, and one that probably never even happened.



Alberto, a hacker, was the perfect target for a judicial system that doesn’t understand cybersecurity or cybercrimes. Through police misconduct and incompetence, his life was thrown into upheaval, and he is still recovering from a system where there are no guarantees of a fair trial.



Rife with injustice and anguish, Alberto’s story doesn’t have a fairy tale ending. It is a lesson for Uruguay and the world to implement fair processes for prosecuting cybercrimes. Alberto’s tale will stand as a stark reminder of the dangers of a judiciary system that hasn’t caught up to the technology of the 21st century.


https://www.linkedin.com/in/nofear75/

Alberto Daniel Hill has four years of experience in the Cryptocurrencies field, three years of experience working with Blockchain technology and has been involved as a team member of an ICO released in 2017. He was later involved as an advisor for another two ICOs.

Alberto is the first hacker in Uruguay who served prison time for a computer related crime. After 4 years of a criminal process, he ended up not having any criminal record. He wrote a book about his story called “OPERACION BITCONS: Login To HELL.

He has been a speaker at several conferences related to Information Security and Fintech in Uruguay and South America, speaking in most of the cases in topics related to cryptocurrencies and security.

Furthermore, he is currently focusing on the Blockchain technology, Crypto Currencies (including Bitcoins) and FinTech sector. After quitting his full-time job as information security professional for the largest company in Uruguay, Alberto founded Crypto Currencies Services in 2016.

Alberto is a computer engineer with more than 20 years of experience linked to Information Security; Consulting, IT security, Computer Forensics, Ethical Hacking, Information Security.

He has a specialization in ISO/IEC-27000, also on Ethical Hacking and approved several courses related to a wide range of IT Security fields such as computer forensics, and an ISO/IEC-20000 specialization. Regarding international certifications, he is PMP-certified and has been leading some Information Security Projects since 2011.

He holds the Cybersecurity Fundamentals Certificate (CSX) from ISACA, an organization that awarded him with a PLATINUM membership recognition.

Also he wrote articles for magazines such as HAKIN9, DIGITAL FORENSICS MAGAZINES and COIN TELEGRAPH. Right now he is completely involved into the BLOCKCHAIN and NFT technology, both in the management of NTF projects and well as focusing on the security in this ecosystem. He also writes articles and teach about these topics, and he is writing two books about NFTs. He is leading a project that is creating the first NFT CLUB for hackers, #HACKER1337

**Comment by XxW1ntermutexX** [comment]
Is this thread still active?
